src/AutoUI/Lenses/LensSelection.tsx

```diff
@@ -21,26 +23,25 @@ export const LensSelection = ({
 	}
 	return (
 		<Box {...rest}>
-			<ButtonGroup>
+			{/* Using `selected` on each item instead of `value` on the group as the Tooltip break the button group logic */}
```

Comment:
I believe you, but could you tell me more about how Tooltip breaks the button group logic? I thought ours simply wraps the component in an additional HTML element